src/modules/homeItemsCounter.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable */
+export const homeItmes = (array) => {
+    if(array.length){
+        return array.length
+    } else {
+       return "0"
+    }
```

Comment:
- [ ] Please know that a counter function should look for specific DOM elements, in this case, the home items, and make the counting based on what is actually displayed on the page, and not based on the array length, kindly fix this.

Reply:
Hi @mirouhml, thanks for the review. We never knew the counter function should look for specific Dom elements. All changes will be implemented